src/System.Management.Automation/help/UpdateHelpCommand.cs

```diff
@@ -347,7 +354,7 @@ internal override bool ProcessModuleWithCulture(UpdatableHelpModuleInfo module,
                     continue;
                 }
 
-                if (Utils.IsUnderProductFolder(module.ModuleBase) && (!Utils.IsAdministrator()))
+                if (Utils.IsUnderProductFolder(moduleBase) && (!Utils.IsAdministrator()) && (this.Scope == UpdateHelpScope.AllUsers))
```

Comment:
> && (this.Scope == UpdateHelpScope.AllUsers)

Is this really needed?

Reply:
Yes, so we only throw an error that elevation is required if we are updating AllUsers scope.

Reply:
The `moduleBase` here won't be under product folder in case that `scope == currentuser`, so no error will be thrown out when `scope == currentuser`.

Reply:
Fixed.